documents/model.go

```diff
@@ -53,6 +69,27 @@ const (
 	ErrZeroCollaborators = errors.Error("require at least one collaborator")
 )
 
+// NewDefaultTree returns a DocumentTree with default opts
+func NewDefaultTree(salts *proofs.Salts) *proofs.DocumentTree {
```

Comment:
I wanted to enforce a single way of creating a tree with the supported defaults. Here is the methods we should be using.